docs/resources/scenarios/punch-clock.md

```diff
@@ -0,0 +1,143 @@
+---
+title: 'Office Scripts sample scenario: Punch clock button'
+description: A sample that lets a user press a button to record the current time.
```

Comment:
```suggestion
description: This sample adds a punch clock button and allows a user to log in and log out using the current time.
```

Reply:
I'll take the spirit of the suggestion, but use "clock" in and out instead of log in, so it doesn't imply an auth path is used.